thoth/storages/graph/models.py

```diff
@@ -1500,6 +1501,64 @@ class KebechetGithubAppInstallations(Base, BaseExtension):
     is_active = Column(Boolean, nullable=False)
 
 
+class SecurityIndicatorAggregatedRun(Base, BaseExtension):
+    """SecurityIndicatorAggregatedRun."""
+
+    __tablename__ = "si_aggregated_run"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    si_aggregated_run_document_id = Column(Text, nullable=False)
+
+    # SI bandit
+    severity_high_confidence_high = Column(Integer, nullable=False)
+    severity_high_confidence_low = Column(Integer, nullable=False)
+    severity_high_confidence_medium = Column(Integer, nullable=False)
+    severity_high_confidence_undefined = Column(Integer, nullable=False)
+    severity_low_confidence_high = Column(Integer, nullable=False)
+    severity_low_confidence_low = Column(Integer, nullable=False)
+    severity_low_confidence_medium = Column(Integer, nullable=False)
+    severity_low_confidence_undefined = Column(Integer, nullable=False)
+    severity_medium_confidence_high = Column(Integer, nullable=False)
+    severity_medium_confidence_low = Column(Integer, nullable=False)
+    severity_medium_confidence_medium = Column(Integer, nullable=False)
+    severity_medium_confidence_undefined = Column(Integer, nullable=False)
+    number_of_analyzed_files = Column(Integer, nullable=False)
+    number_of_files_total = Column(Integer, nullable=False)
+    number_of_files_with_severities = Column(Integer, nullable=False)
+    number_of_filtered_files = Column(Integer, nullable=False)
+
+    # SI cloc
+
+    ## Python files
+    number_of_python_files = Column(Integer, nullable=False)
+    number_of_lines_with_comments_in_python_files = Column(Integer, nullable=False)
+    number_of_blank_lines_in_python_files = Column(Integer, nullable=False)
+    number_of_lines_with_code_in_python_files = Column(Integer, nullable=False)
+
+    ## All files
+    total_number_of_files = Column(Integer, nullable=False)
+    total_number_of_lines = Column(Integer, nullable=False)
+    total_number_of_lines_with_comments = Column(Integer, nullable=False)
+    total_number_of_blank_lines = Column(Integer, nullable=False)
+    total_number_of_lines_with_code = Column(Integer, nullable=False)
+
+    python_package_version_entities = relationship("Aggregated", back_populates="si_aggregated_run")
+
+
+class Aggregated(Base, BaseExtension):
```

Comment:
Worth to prefix with SI or something like that. I bet we will have more "aggregated-type" results.